thorlcr/graph/thgraph.hpp

```diff
@@ -1194,7 +1199,7 @@ interface IThorFileCache : extends IInterface
     virtual IFileIO *lookupIFileIO(CActivityBase &activity, const char *logicalFilenae, IPartDescriptor &partDesc, IExpander *expander=nullptr) = 0;
 };
 
-extern graph_decl IDelayedFile *createDelayedFile(IFileIO *iFileIO);
+extern graphslave_decl IDelayedFile *createDelayedFile(IFileIO *iFileIO);
```

Comment:
@jakesmith you may want to move this function to a slave header file.  For the moment I just changed the declspec

Reply:
@ghalliday - I'll issue a JIRA/PR to move it into thgraphslave.hpp

Reply:
JIRA for reference : https://track.hpccsystems.com/browse/HPCC-24924